function/metadata.go

```diff
@@ -23,15 +23,13 @@ import (
 
 	"github.com/BurntSushi/toml"
 	"github.com/buildpack/libbuildpack/application"
-	"github.com/cloudfoundry/libcfbuildpack/helper"
-	"github.com/cloudfoundry/libcfbuildpack/logger"
+	"github.com/buildpack/libbuildpack/logger"
 )
 
 const (
-	RiffEnv     = "RIFF"
-	ArtifactEnv = "RIFF_ARTIFACT"
-	HandlerEnv  = "RIFF_HANDLER"
-	OverrideEnv = "RIFF_OVERRIDE"
+	ArtifactEnv = "ARTIFACT"
```

Comment:
If these constants are package public they should have descriptive comments.  Do they need to be public?